dip-template/nodes/dip-consumer/src/command.rs

```diff
@@ -270,14 +270,6 @@ pub fn run() -> Result<()> {
 					if config.role.is_authority() { "yes" } else { "no" }
 				);
 
-				if !collator_options.relay_chain_rpc_urls.is_empty() && cli.relay_chain_args.is_empty() {
```

Comment:
Why did you remove this?

Reply:
`CollatorOptions` no longer exposes the relay_chain_rpc_urls. Therefore, it cannot be checked whether it is empty or not. I verified this with the parachain template, and the check has been removed there as well.